src/styles.scss

```diff
@@ -1230,3 +1230,21 @@ $truncate-text-values: 1, 2, 3;
 .mat-header-cell:not(:last-of-type) {
   padding-right: 1.6rem;
 }
+
+// This style should be used for buttons and drop downs in the private "My Dockstore" pages, they do not seem to exist in public dockstore pages
+// These redesigned buttons have purple text and white backgrounds
+button.private-btn {
+  font-weight: bold;
```

Comment:
The private button has bolded text, but the public Info button doesn't. Should they both be bolded?

Reply:
The public Info button will be bolded eventually, it falls under the mat-raised-button style which will have a global change with this [ticket](https://github.com/dockstore/dockstore/issues/4680) to update them with bolded text and etc. So to avoid redundant styling it'll be left so until then